### PR TITLE
Center tag icon

### PR DESCRIPTION
**Pull request by aa0-fyi**
Resolve issue #504.

### CODE REVIEW

**Review by ericfennis — CHANGES_REQUESTED** [review]
You moved the icon a bit. What feels like the solution but there is problem by doing this. By centering the icon the path will add some decimals to some points. If rendered on low pixel displays the icon will appear blurry. (See screenshot, left is yours). We can fix this by keep the points rounded. I fixed it by extended the `tag` icon so it centered again, but keeping the icon pixel perfect.

Screenshot 2022-03-15 at 19 43 15

File: icons/tag.svg [hunk 1]
```diff
@@ -9,6 +9,6 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <path d="M20.59 13.41l-7.17 7.17a2 2 0 01-2.83 0L2 12V2h10l8.59 8.59a2 2 0 010 2.82z" />
-  <line x1="7" y1="7" x2="7.01" y2="7" />
+  <path d="m21.007 13.83-7.17 7.17a2 2 0 0 1-2.83 0l-8.59-8.58v-10h10l8.59 8.59a2 2 0 0 1 0 2.82z"/>
+  <line x1="7.417" x2="7.427" y1="7.42" y2="7.42"/>
```
[the remark this answers is not in the document]
Comment (aa0-fyi) [reply icons/tag.svg:13]: Excellent catch.

[the remark this answers is not in the document]
Comment (aa0-fyi) [reply icons/tag.svg]: A new problem appears to be the left-center point of the outside of the tag. That point is no longer rounded on full resolution. I attempted to resolve this but was unsuccessful while maintaining pixel perfection.

[the remark this answers is not in the document]
Comment (ericfennis) [reply icons/tag.svg]: I think this is the best solution. Most important thing is that the icon not look blurry on low resolution screens

**Pull request merged**